packages/faucet/database-rules.bolt

```diff
@@ -57,23 +89,3 @@ isAllowed(ref) {
   return auth.uid == "gYOD4GPV86OlqQh33Loc5lh0Yo02" || auth.uid == "LLnGFlilnxbXrb5pZzvjiDeUJ6l2"
```

Comment:
i thought you had written that showing these was a security issue?  did i misread?

Reply:
It's not ideal but not exactly an issue. This is the whitelist, and it'd be better to keep them secret, but they're out there already. In theory, there's no way for a user to impersonate these uid except by logging in with our secret creds